### ISSUE TITLE
bitbake openvario-image-testing fails because sox is missing

### ISSUE THREAD

**Issue by bomilkar** [Openvario/meta-openvario#323]
Recent versions of xcsoar require sox. Here is a snippet of the error message when `bitbake openvario-image-testing`

```
|   SOX     output/data/sound/beep_bweep.raw
| make: sox: No such file or directory
| make: *** [build/resource.mk:156: output/data/sound/beep_bweep.raw] Error 127
```
Please note `bitbake openvario-image` doesn't need sox and therefore is not affected.

**Comment by Scumi** [comment]
Should we then add "--pull=always" to the docker command in the description?

**Comment by linuxianer99** [comment]
Think this generates unnecessary load ... 

In my mind, i would have a look in the issues, if somebody has reported the problem already and if there is a resolution, before creating a new issue ...

**Comment by bomilkar** [comment]
I'm sorry when I haven't seen that the issue had been reported already. Out of curiosity: how could I have found it??

Here is what I did when bitbake failed:
I built xcsoar to realize sox was missing. The developers manual mentions sox under the Requirements. Adding the package is straight forward in the native linux environment.
However, the docker system seems quite convoluted to me. I noticed "which sox" didn't find anything and poking around where to add it didn't lead nowhere.

@Scumi : thanks for the **constructive** hint! Adding "--pull=always" does the trick (and it didn't add significant load).
